db/migrate/20230401114456_make_terms_translatable.rb

```diff
@@ -0,0 +1,15 @@
+class MakeTermsTranslatable < ActiveRecord::Migration[6.1]
+  def up
+    rename_column :documents, :title, :title_translations
```

Comment:
@franpb14 this migration crashes for me locally:

```
== 20230401114456 MakeTermsTranslatable: migrating ============================
-- rename_column(:documents, :title, :title_translations)
   -> 0.0069s
-- rename_column(:documents, :content, :content_translations)
   -> 0.0030s
-- change_column(:documents, :title_translations, :jsonb, {:default=>{}, :null=>false, :using=>"title_translations::jsonb"})
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

PG::InvalidTextRepresentation: ERROR:  invalid input syntax for type json
DETAIL:  Token "Terminos" is invalid.
CONTEXT:  JSON data, line 1: Terminos...
/Users/markets/workspace/timeoverflow/db/migrate/20230401114456_make_terms_translatable.rb:5:in `up'
```

I think a much robust approach is:
- create new cols
- migrate data
- remove old cols